src/platforms/flutter/config.yml

```diff
@@ -1,3 +1,8 @@
 title: Flutter
-caseStyle: camelCase
+caseStyle: canonical
```

Comment:
Why did this change? Sure this is right?

Options shown as:
![image](https://user-images.githubusercontent.com/1633368/100269712-f3c47a00-2f24-11eb-974b-55d0b0b5c46e.png)